pandas/tests/indexes/test_base.py

```diff
@@ -106,6 +106,16 @@ def test_construction_list_mixed_tuples(self):
         assert isinstance(idx2, Index)
         assert not isinstance(idx2, MultiIndex)
 
```

Comment:
make this is a new test.

parametrize on input types (iterator, tuple, list) & (np.nan, None) for a missing value

Reply:
(done)

@jreback ping